build/lib/release.sh

```diff
@@ -466,39 +471,99 @@ function kube::release::package_kube_manifests_tarball() {
   kube::release::create_tarball "${package_name}" "${release_stage}/.."
 }
 
+# Builds tarballs for each test platform containing the appropriate binaries.
+function kube::release::package_test_platform_tarballs() {
```

Comment:
I'm just marveling at the fact that you can have colons in a function name in BASH. I had no idea.

Reply:
There's some really amazing bash in this project. And also plenty of horrifying stuff #72956 😛